src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -480,7 +480,7 @@ def _to_vespa_lexical_query(self, marqo_query: MarqoLexicalQuery) -> Dict[str, A
             query_inputs.update(score_modifiers)
 
         query = {
-            'yql': f'select {select_attributes} from {self._marqo_index.schema_name} where {lexical_term}{filter_term}',
+            'yql': f'select {select_attributes} from {self._marqo_index.schema_name} where {search_term}',
```

Comment:
The change here is to wrap lexical_term and filter_term in parentheses. Is this a bug fix? Is there test cases to cover this?

Reply:
I think no parentheses wouldn't have been an issue with weakAnd because you would get weakAnd(... OR ..., ... OR ...) AND filter